### ISSUE TITLE
add support for shortcut

### ISSUE THREAD

**Issue by AdamPype** [vamsi3/screen-translator#12]
hello! thank you so much for this wonderful app. it's making my life in korea much easier :) one thing that would be nice, is adding support for the shortcut feature. right now when I set a shortcut in the accessibility settings and try the shortcut, nothing happens, and the permission is reset.

![Screenshot_20230511_133844_Accessibility.jpg](https://github.com/vamsi3/screen-translator/assets/43540805/9676dfb8-6c65-42d1-ad1c-655f6f3f3d9a)

**Comment by vamsi3** [comment]
Yes, this is a feature that I tried to add, but on my Samsung S23 Ultra, accessibility button behaviour doesn't seen to the same as say a Google Pixel. I ran into some issues and never came back at it.

Let me give it a shot again and see if I can get it to work. What phone are you on?

**Comment by AdamPype** [comment]
Same one as you!

Would love to have it working :) Thank you so much!

**Comment by MohamadBayz** [comment]
Here The shortcut isn't working either i have Mi 9T running android 13 with AOSP Based Rom
When running the shortcut it disables the screen translator's permission for accessibility like in the recording 
Note I'm not disabling it myself it disables itself when i swipe up with two fingers on the navigation bar

https://github.com/vamsi3/screen-translator/assets/63660736/7a75ac0a-2719-4a51-ad93-cba167e247e4

**Comment by AdamPype** [comment]
@vamsi3 hey! any update on this? just bumping this :)
